document/zh/deploy_radondb-clickhouse_operator_on_kubernetes.md

```diff
@@ -0,0 +1,170 @@
+
+本教程演示如何使用命令行在 Kubernetes 上部署 RadonDB ClickHouse。
+
+## 部署准备
+
+- 已成功部署 Kubernetes 集群。
+- 已安装 Helm 包管理工具。
+
+## 部署步骤
+
+### 步骤 1 : 添加仓库
+
+添加并更新 Helm 仓库。
+
+```bash
+$ helm repo add ck https://radondb.github.io/radondb-clickhouse-kubernetes/
+$ helm repo update
+```
+
+### 步骤 2 : 部署 RadonDB ClickHouse Operator
+
+```bash
+$ helm install clickhouse-operator ck/clickhouse-operator
+```
+
+### 步骤 3 : 部署 RadonDB ClickHouse 集群
+
+```bash
+$ helm install clickhouse ck/clickhouse-cluster
+```
+
+### 步骤 4 : 部署校验
+
+#### 查看 Pod 运行状态
+
+执行如下命令，查看创建的集群 Pod 运行状态。
+
+```bash
+kubectl get pods -n <Namespace>
+```
+
+**预期结果**
+
+```shell
+$ kubectl get pods -n test
+NAME                                READY   STATUS    RESTARTS   AGE
+pod/chi-ClickHouse-replicas-0-0-0   2/2     Running   0          3m13s
+pod/chi-ClickHouse-replicas-0-1-0   2/2     Running   0          2m51s
+pod/chi-ClickHouse-replicas-1-0-0   2/2     Running   0          2m34s
+pod/chi-ClickHouse-replicas-1-1-0   2/2     Running   0          2m17s
+pod/chi-ClickHouse-replicas-2-0-0   2/2     Running   0          115s
+pod/chi-ClickHouse-replicas-2-1-0   2/2     Running   0          48s
+pod/zk-clickhouse-cluster-0         1/1     Running   0          3m13s
+pod/zk-clickhouse-cluster-1         1/1     Running   0          3m13s
+pod/zk-clickhouse-cluster-2         1/1     Running   0          3m13s
+```
+
+#### 查询 SVC 运行状态
+
+执行如行命令，查看集群 SVC 运行状态。
+
+```bash
+$ kubectl get service -n <Namespace>
+```
+
+**预期结果**
+
+```shell
+$ kubectl get service -n test
+NAME                                  TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                         AGE
+service/chi-ClickHouse-replicas-0-0   ClusterIP   None            <none>        8123/TCP,9000/TCP,9009/TCP      2m53s
+service/chi-ClickHouse-replicas-0-1   ClusterIP   None            <none>        8123/TCP,9000/TCP,9009/TCP      2m36s
+service/chi-ClickHouse-replicas-1-0   ClusterIP   None            <none>        8123/TCP,9000/TCP,9009/TCP      2m19s
+service/chi-ClickHouse-replicas-1-1   ClusterIP   None            <none>        8123/TCP,9000/TCP,9009/TCP      117s
+service/chi-ClickHouse-replicas-2-0   ClusterIP   None            <none>        8123/TCP,9000/TCP,9009/TCP      50s
+service/chi-ClickHouse-replicas-2-1   ClusterIP   None            <none>        8123/TCP,9000/TCP,9009/TCP      13s
+service/clickhouse-ClickHouse         ClusterIP   10.96.137.152   <none>        8123/TCP,9000/TCP               3m14s
+service/zk-client-clickhouse-cluster  ClusterIP   10.107.33.51    <none>        2181/TCP,7000/TCP               3m13s
+service/zk-server-clickhouse-cluster  ClusterIP   None            <none>        2888/TCP,3888/TCP               3m13s
+```
+
+## 访问 RadonDB ClickHouse
+
+### 通过 Pod
+
+通过 `kubectl` 工具直接访问 ClickHouse Pod。
+
+```bash
+kubectl exec -it <pod name> -n <project name> -- clickhouse-client --user=<user name> --password=<user password>
```

Comment:
should be better to use `-n <Namespace>` to replace `-n <project name>`, so that it can be consistent with the previous

Reply:
DONE